core/kernel/dt.c

```diff
@@ -449,6 +450,11 @@ void *get_dt(void)
 	if (!fdt)
 		fdt = get_external_dt();
 
+#if defined(CFG_CORE_FFA)
```

Comment:
1. Drop the `#if`, is it not needed.
2. Shouldn't `get_manifest_dt()` be declared in `include/kernel/dt.h` and implemented here? TBD in a separate commit prior to this one.

Reply:
1. Dropped `#if`
2. Added another commit to move manifest DT functions to kernel/dt.c